src/components/ButtonMenu.tsx

```diff
@@ -21,13 +21,14 @@ interface ButtonMenuProps {
   placement?: "left" | "right";
   // for storybook purposes
   defaultOpen?: boolean;
+  isDisabled?: boolean;
```

Comment:
Minor, but consistency we typically use `disabled`; granted the react-aria API we use internally for some implementation details uses `isDisabled` so you'll see renames from our public API `disabled` --> `isDisabled` where we cross the beam/react-aria API boundaries.

Reply:
thanks for the insight - I'll update